internal/service/clusteroutagesimulation/resource_cluster_outage_simulation.go

```diff
@@ -30,7 +30,7 @@ func Resource() *schema.Resource {
 		UpdateContext: resourceUpdate,
 		DeleteContext: resourceDelete,
 		Timeouts: &schema.ResourceTimeout{
-			Delete: schema.DefaultTimeout(25 * time.Minute),
+			Delete: schema.DefaultTimeout(40 * time.Minute),
```

Comment:
This change must probably be reverted (only to make the Acc tests pass)

Reply:
Passing acceptance tests:
1. migration tests failing due to DUPLICATE_CLUSTER_NAME  <https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/9938653698/job/27451730157#step:5:77>
2. migration tests failing due to timeout <https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/9938653698/job/27451730157#step:5:77>

Reply:
Tests are passing!